public/pages/login.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style/style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Find My Musician.com</title>
+</head>
+
+<body>
+    <header>
+        <div class="container">
+        <h1>Find My Musician.com </h1>
+        <nav>
+            <ul>
```

Comment:
you can copy the header from the index.html so they all are they same and have the same ids.